third_party/nvfuser/csrc/python_frontend/python_bindings.cpp

```diff
@@ -301,6 +302,7 @@ void initNvFuserPythonBindings(PyObject* module) {
                   " is not supported in nvFuser. Expected size > 0.");
               if (sizes[i] == 1) {
                 maybe_symbolic_sizes.push_back(1);
+                contiguity.erase(contiguity.begin() + i);
```

Comment:
Strange. Isn't this already handled by `computeContiguity` https://github.com/csarofeen/pytorch/blob/976b43c41da9e32c735beb775f22cdb5521b3c90/third_party/nvfuser/csrc/python_frontend/python_bindings.cpp#L28

Reply:
Ahh, I see. I am OK with changing `not_broadcast` for now to unblock. But I think a better solution is to a some refactor on the python frontend to handle size N stride 0 dimension as expanded broadcast.

Reply:
#2561 handles size N stride 0. Should we move along with that instead of this PR?